include/fastdds/rtps/messages/RTPSMessageSenderInterface.hpp

```diff
@@ -77,28 +77,16 @@ class RTPSMessageSenderInterface
         /**
          * Send a message through this interface.
          *
-         * @param message Pointer to the buffer with the message already serialized.
+         * @param buffers Array of buffers to gather.
+         * @param num_buffers Number of elements on @c buffers.
+         * @param total_bytes Total size of the raw data. Should be equal to the sum of the @c length field of all buffers.
          * @param max_blocking_time_point Future timepoint where blocking send should end.
          */
         virtual bool send(
                 const NetworkBuffer* message,
```

Comment:
The doxygen parameter name should be the same as the one declared in the function.

```suggestion
                const NetworkBuffer* buffers,
```

Reply:
Fixed in dc3799349c14b248e1aa6634ca9f354db375f42a